openmetrics/src/main/java/io/airlift/openmetrics/JmxOpenMetricsModule.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.openmetrics;
+
+import com.google.inject.Binder;
+import com.google.inject.Module;
+
+import javax.management.MBeanServer;
+
+import java.lang.management.ManagementFactory;
+
+import static io.airlift.configuration.ConfigBinder.configBinder;
+import static io.airlift.jaxrs.JaxrsBinder.jaxrsBinder;
+
+public class JmxOpenMetricsModule
+        implements Module
+{
+    @Override
+    public void configure(Binder binder)
+    {
+        binder.disableCircularProxies();
+
+        binder.bind(MBeanServer.class).toInstance(ManagementFactory.getPlatformMBeanServer());
```

Comment:
This doesn't depend on JmxModule, it can be used with or without it.

Reply:
It might be detecting that the object returned by `ManagementFactory.getPlatformMBeanServer()` is the same instance, but that's quite brittle. 

Indeed, this module should not bind the MBeanServer -- that's the job of JmxModule. This module should just be in charge of wiring the REST resource, config, etc.

Reply:
It will always be the same instance, it's even documented in the jdk to be the same instance. https://docs.oracle.com/en/java/javase/18/docs/api/java.management/java/lang/management/ManagementFactory.html#getPlatformMBeanServer() I'll look further next week if you feel strongly about this.

Reply:
This has been a part of this PR for 4 months, so if it hasn't been a priority in that timeframe, then I don't see it as a priority to resolve. I will get to this and the other feedback in the coming weeks. Please take time to review this fully and I'll resolve as I have time.